src/chrome/content/rules/TryMagPop.ca.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="TryMagPop.ca" platform="mixedcontent">
```

Comment:
I only see mixed fonts, but we don't consider them as "breaking".

Reply:
Fixed, thanks!